docs/source/utils.rst

```diff
@@ -1,21 +1,20 @@
+.. py:module:: torchaudio.utils
+
 torchaudio.utils
 ================
 
-.. py:module:: torchaudio.utils
-
-torchaudio.utils.sox_utils
-~~~~~~~~~~~~~~~~~~~~~~~~~~
+``torchaudio.utils`` module contains utility functions to configure the global state of third party libraries.
 
-Utility module to configure libsox.
-This affects functionalities in :ref:`Sox IO backend<sox_io_backend>` and :ref:`Sox Effects<sox_effects>`.
+py:mod:`~torchaudio.utils.sox_utils` affects functionalities in :ref:`Sox IO backend<sox_io_backend>` and :ref:`Sox Effects<sox_effects>`.
```

Comment:
This is not output correctly https://output.circle-artifacts.com/output/job/bcd411e1-349b-48c0-a4ac-b8f704fec6ef/artifacts/0/docs/utils.html